canvas_modules/common-canvas/src/common-properties/panels/subtabs/subtabs.scss

```diff
@@ -83,4 +82,8 @@ $selected-tab-width: 3px;
 		width: 100%;
 		padding: $spacing-05; // Override padding set by subtab
 	}
+
+	div.cds--tabs:not(.cds--tabs--contained) .cds--tabs__nav-item:not(.cds--tabs__nav-item--selected):not(.cds--tabs__nav-item--disabled):hover {
```

Comment:
I don't usually like doing this might this might be a case where it's best to us !important.  So instead of this you can have
```
		button.properties-leftnav-subtab-item {
			&:hover {
				background-color: $layer-hover-01;
				border-block-end: none !important;
			}
```
Just make sure to add a comment to why we have it.